tool/rettag_v1.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
Nit: Can we provide a file documentation that explains when this should be run and how? 👀